content/en/docs/concepts/configuration/resource-bin-packing.md

```diff
@@ -0,0 +1,197 @@
+
+```json
+{
+    "kind" : "Policy",
+    "apiVersion" : "v1",
+
+    ...
+
+    "priorities" : [
+
+       ...
+
+      {
+        "name": "RequestedToCapacityRatioPriority",
+        "weight": 2,
+        "argument": {
+          "requestedToCapacityRatioArguments": {
+            "shape": [
+              {"utilization": 0, "score": 0},
+              {"utilization": 100, "score": 10}
+            ],
+            "resources": [
+              {"name": "intel.com/foo", "weight": 3},
+              {"name": "intel.com/bar", "weight": 5}
+            ]
+          }
+        }
+      }
+    ],
+  }
+```
+
+**This feature is disabled by default**
+
+### Tuning RequestedToCapacityRatioResourceAllocation Priority Function
+
+`shape` is used to specify the behavior of the `RequestedToCapacityRatioPriority` function.
+
+```yaml
+ {"utilization": 0, "score": 0},
+ {"utilization": 100, "score": 10}
+```
+
+The above arguments give the node a score of 0 if utilization is 0% and 10 for utilization 100%, thus enabling bin packing behavior. To enable least requested the score value must be reversed as follows.
+
+```yaml
+ {"utilization": 0, "score": 100},
+ {"utilization": 100, "score": 0}
+```
+
+`resources` is an optional parameter which by defaults is set to:
+
+``` yaml
+"resources": [
+              {"name": "CPU", "weight": 1},
+              {"name": "Memory", "weight": 1}
+            ]
+```
+
+It can be used to add extended resources as follows: 
+
+```yaml
+"resources": [
+              {"name": "intel.com/foo", "weight": 5},
+              {"name": "CPU", "weight": 3},
+              {"name": "Memory", "weight": 1}
+            ]
+```
+
+The weight parameter is optional and is set to 1 if not specified. Also, the weight cannot be set to a negative value.
+
+### How the RequestedToCapacityRatioResourceAllocation Priority Function Scores Nodes
+
+This section is intended for those who want to understand the internal details
+of this feature.
+Below is an example of how the node score is calculated for a given set of values.
+
+```
+Requested Resources
+
+intel.com/foo : 2
+Memory: 256MB
+CPU: 2
+
+Resource Weights
+
+intel.com/foo : 5
+Memory: 1
+CPU: 3
+
+FunctionShapePoint {{0, 0}, {100, 10}}
+
+Node 1 Spec
+
+Available:
+intel.com/foo : 4
+Memory : 1 GB
+CPU: 8
+
+Used:
+intel.com/foo: 1
+Memory: 256MB
+CPU: 1
+
+
+Node Score:
+
+intel.com/foo  = resourceScoringFunction((2+1),4)
+               =  (100 - ((4-3)*100/4)
+               =  (100 - 25)
+               =  75
+               =  rawScoringFunction(75)
+               = 7
+
+Memory         = resourceScoringFunction((256+256),1024)
+               = (100 -((1024-512)*100/1024))
+               = 50
+               = rawScoringFunction(50)
+               = 5
+
+CPU            = resourceScoringFunction((2+1),8)
+               = (100 -((8-3)*100/8))
+               = 37.5
+               = rawScoringFunction(37.5)
+               = 4
```

Comment:
37 will actually be mapped to 3.

Reply:
yea that correct 👍 will update the calculation, thanks for catching it.